src/modules/singefileContext.js

```diff
@@ -0,0 +1,17 @@
+import { AbstractProjectContext } from './abstractProjectContext.js';
+import { SessionManager } from './sessionManager.js';
+
+class SinglefileContext extends AbstractProjectContext {
+
+    /**
+     * Create a context for a specific project. This can be used to search in a project or list its images.
```

Comment:
I believe hansken.py doesn't make a difference between a `SinglefileContext` and a `ProjectContext` at all. Not sure if we want the same pattern here. The only difference between projects and singlefiles in hansken.py is the upload call I believe.

Reply:
```suggestion
     * Create a context for a singlefile. This can be used to search in it.
```